frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/CheckReAuthUserHandler.java

```diff
@@ -181,6 +183,7 @@ private APIGatewayProxyResponseEvent generateSuccessResponse() {
     private APIGatewayProxyResponseEvent generateErrorResponse(
             String email, AuditContext auditContext) {
         if (hasEnteredIncorrectEmailTooManyTimes(email)) {
+            if (configurationService.supportReauthSignoutEnabled()) removeEmailCountLock(email);
```

Comment:
```suggestion
            if (configurationService.supportReauthSignoutEnabled()) {
                removeEmailCountLock(email);
            }
```

Reply:
Do we have tests for enabled and disabled?

Reply:
No, we have not, considering that in the current test we only check for the response that the lambda gives back I don't think it's necessary. I believe this will change when we have the full scope of this feature, with the audit events and error messages that need to be added in the short future.